src/mmw/js/src/modeling/gwlfe/runoff/views.js

```diff
@@ -154,6 +154,16 @@ function monthSorter(a, b) {
 
 window.monthSorter = monthSorter;
 
+function SumFormatter(data) {
+    var field = this.field,
+        total = data.reduce(function(sum, row) {
+            return sum + (parseFloat(row[field]) || 0);
+        }, 0);
```

Comment:
Consider turning this into a map and a sum operation, which might be easier to read.